README.md

```diff
@@ -1 +1,53 @@
-# MyPortfolio
\ No newline at end of file
+# Portfolio
+
+> My Personal Portfolio
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**Run HTML file**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+**pass Linters Tests**
+
+### Deployment
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your README file to make it more professional by improving the project description, content in some sections(such as the **Getting Started** section,...), removing the empty and non-related sections,... To make it easy for you just take a look at this **[README Template](https://github.com/tresorsawasawa/MyPortfolio#readme)** then complete your README to be similar to it.